### ISSUE TITLE
Node 8 LTS support?

### ISSUE THREAD

**Issue by tc** [aws/aws-codebuild-docker-images#45]
Elastic Beanstalk has a node 8  support, will there be a node 8 build for codebuild?

I only see node 6.3.1 as the latest.

**Comment by andrewholsted** [comment]
Agreed. Looking for Node 8 support and would prefer not to have to create my own AWS ECR for it

**Comment by jfdoube** [comment]
AWS lambda now support nodejs 8.10 and we cannot yet leverage it because code build does not support 8.10 yet. Please make it GA
